src/Adapter/DbSelect.php

```diff
@@ -17,7 +17,7 @@
 
 class DbSelect implements AdapterInterface
 {
-    const ROW_COUNT_COLUMN_NAME = 'C';
+    const ROW_COUNT_COLUMN_NAME = 'c';
```

Comment:
There is another attribute: `PDO::CASE_UPPER` – force column names to upper case

Reply:
that's covered at https://github.com/laminas/laminas-paginator/pull/11 but still not testable yet. I will need to back switch to php 7.3 locally to avoid Reflection ::__string error instead of pushing commit and let travis run the test